src/components/accordion/Accordion.js

```diff
@@ -84,14 +84,19 @@ export class LeuAccordion extends LeuElement {
         <span class="label">${this.label}</span>
         <div class="plus"></div>
       </button></${unsafeStatic(hTag)}>
-      <div
-        id="content"
-        class="content"
-        aria-labelledby="toggle"
-        role="region"
+      <div 
+        id="contentwrapper"
```

Comment:
This id is never referenced. You probably should change the value of the aria-controls attribute to this id.

Move the following attributes of the "content" element to the "contentwrapper" element:
- `aria-labelledby="toggle"`
- `role="region"`

Reply:
done